spec/models/dialog_spec.rb

```diff
@@ -61,13 +62,13 @@
 
   context "#create" do
     it "validates_presence_of name" do
-      expect { FactoryGirl.create(:dialog) }.to raise_error
+      expect { FactoryGirl.create(:dialog) }.to raise_error(ActiveRecord::RecordInvalid, /Label can't be blank/)
```

Comment:
Indeed, I'd tend to agree :confused:

I'm unfamiliar with these classes and what would be required to make it valid; one consideration when making every factory valid by default is that you could get into some nasty slow creations with giant dependency trees (as a factory might need X via association, which in turn needs Y via association, etc). It gets ugly fast, and in those situations, fixtures (instead of factories) are much more appropriate.

At the very least these should be named more appropriately given that they're invalid (`:invalid_dialog`, something)

But I digress, that's for another time.